src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -783,13 +783,9 @@ public int CompareTo(String strB)
         }
 
         [Pure]
-        internal bool EndsWith(char value) {
+        public Boolean EndsWith(char value) {
```

Comment:
Nit: `Boolean` -> `bool`. I know this file is inconsistent, but at some point soon it's going to be cleaned up to follow the CoreFX coding style where language keywords are preferred over BCL types.